<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flashcards</class>
 <widget class="QMainWindow" name="Flashcards">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flashcards</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>521</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Solve Quiz</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>361</width>
        <height>134</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="answer_button_1">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="answer_button_2">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="answer_button_3">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="answer_button_4">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>258</width>
        <height>218</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Add question</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>358</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="add_question_widget">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_question"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_correct_answer">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>356</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="wrong_answers">
         <property name="text">
          <string>Wrong answers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_wrong_answer_1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="text_wrong_answer_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="text_wrong_answer_3"/>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
